### PR TITLE
Toast notifications

### PR DESCRIPTION
**Pull request by agrim123**
Enabled toast notifications using [toastr](https://github.com/CodeSeven/toastr) instead of previously used bootstrap flash messages.
Includes both flash messages and new toastr messages for comparison.
The toast notification shown can be significantly styled according to needs and convenience. :smile: 

Addresses #1331

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -637,4 +637,4 @@ DEPENDENCIES
   whenever
 
 BUNDLED WITH
-   1.14.3
+   1.14.5
```
Comment (ChrisBr) [inline Gemfile.lock]: This is unrelated to your changes!

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -148,6 +148,36 @@ def bootstrap_class_for(flash_type)
...
+            'timeOut': '5000',
+            'extendedTimeOut': '1000',
+            'showEasing': 'swing',
+            'hideEasing': 'linear',
+            'showMethod': 'fadeIn',
+            'hideMethod': 'fadeOut'
+          };
+          toastr.#{type}('#{message}');
+      </script>"
+      flash_messages << text.html_safe if message
```
Comment (ChrisBr) [inline app/helpers/application_helper.rb]: Don't do that! You also make use ``html_safe`` on the message! However, message can be a user input which leads to a XSS vulnerability!

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -6,6 +6,7 @@
     %meta{:content => "", :name => "description"}
     %meta{:content => "", :name => "author"}
     = stylesheet_link_tag "application", :media => "all"
+    = stylesheet_link_tag "https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.css"
```
Comment (ChrisBr) [inline app/views/layouts/application.html.haml]: Why not using the gem? 
https://github.com/d4be4st/toastr_rails

File: app/views/layouts/application.html.haml [hunk 4]
```diff
@@ -18,10 +19,11 @@
       };
     = content_for(:script_head)
     = javascript_include_tag "//cdn.transifex.com/live.js"
-
+    = javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"
```
Comment (ChrisBr) [inline app/views/layouts/application.html.haml]: Why not using the gem?
https://github.com/d4be4st/toastr_rails

**Comment by ChrisBr** [comment]
@agrim123 In general: Please don't open an issue for a feature request and immediately start implementing it without waiting for any feedback. 

> Addresses #1331

Sounds to me like there is already an agreement on implementing it. However, while reviewing #1331 I noticed that you opened the issue 20 hours ago and didn't wait for any feedback...

**Comment by ChrisBr** [comment]
@differentreality @Ana06 @lagartoflojo what do you think about this?

**Comment by ChrisBr** [comment]
Please also include screenshots!

**Comment by agrim123** [comment]
@ChrisBr 

- Screenshot of toast appearing when success is populated
toast-success
- Screenshot of toast appearing when error is popullated 
toast-error

**Comment by differentreality** [comment]
@ChrisBr I really do not know if we want this, I am closing this until we decide on the issue.
